intermediate_source/torch_export_tutorial.py

```diff
@@ -0,0 +1,497 @@
+# -*- coding: utf-8 -*-
+
+"""
+torch.export Tutorial
+================
+**Author:** William Wen, Zhengxu Chen
+"""
+
+######################################################################
+# :func:`torch.export` is the PyTorch 2.0 way to export PyTorch models into
```

Comment:
nit: "PyTorch 2," not "PyTorch 2.0"

Reply:
@gchanan we are using `PyTorch 2.0` in a lot of places as title for poster etc.: https://events.linuxfoundation.org/pytorch-conference/program/schedule/ is that OK?

Reply:
Could we say 2.x so that it stays relevant to all 2.x versions?